### PR TITLE
feat(android): support hiding Exoplayer video duration on android

### PR DESCRIPTION
**Pull request by ashlyWeiting**
## Summary
This PR adds support for hiding the duration on Android devices.

### Changes
  -  Implemented a new prop to hide the duration on Android.
  -  Updated relevant Android-specific components to respect the new prop.

## Test plan
    1. Apply the new prop hideDuration = true to hide the duration.
    2. Verify that the video player does not display the duration when the prop is true.

### CODE REVIEW

**Comment by seyedmostafahasani** [comment]
@ashlyWeiting 
I don't understand why we need this PR. In particular, when the user hides the seek bar, it indicates that the video is live, so we hide both the seek bar and the duration. However, if the user only wants to hide the duration, it might cause confusion.
Why would the user need to hide the duration?

**Comment by ashlyWeiting** [comment]
I know this request is unusual, but over 10 Android users of my product have asked for it. Users think seeing the current position is sufficient. 
If you think this PR is unnecessary, I will close it.

Thank you for taking time to review my pull request!

**Comment by seyedmostafahasani** [comment]
@freeboub @KrzysztofMoch 
WDYT?

**Comment by freeboub** [comment]
I think this is not up to rnv to choose UI requirements. Btw, the pr is clean and easy to maintain, so ok for me !

**Review by freeboub — CHANGES_REQUESTED** [review]
I just think there is a missing typescript change in the component file. Can you double check please?

**Comment by seyedmostafahasani** [comment]
@ashlyWeiting 
Please add the missing TypeScript to this file `src/specs/VideoNativeComponent.ts`

```
type ControlsStyles = Readonly<{
  hideSeekBar?: boolean;
  hideDuration?: boolean;
  seekIncrementMS?: Int32;
}>;
```

File: src/types/video.ts [hunk 1]
```diff
@@ -223,6 +223,7 @@ export type AudioOutput = 'speaker' | 'earpiece';
 
 export type ControlsStyles = {
   hideSeekBar?: boolean;
+  hideDuration?: boolean;
```
Comment (seyedmostafahasani) [inline src/types/video.ts:226]: Please add the missing TypeScript to this file `src/specs/VideoNativeComponent.ts`
```
type ControlsStyles = Readonly<{
  hideSeekBar?: boolean;
  hideDuration?: boolean;
  seekIncrementMS?: Int32;
}>;
```

**Comment by seyedmostafahasani** [comment]
@ashlyWeiting 
Please update the document accordingly.

**Comment by ashlyWeiting** [comment]
Thanks for the review! I've made the updates you requested. Could you please take a look and let me know if everything looks good?

**Review by freeboub — APPROVED** [review]

**Review by seyedmostafahasani — APPROVED** [review]

**Comment by seyedmostafahasani** [comment]
@KrzysztofMoch 
I think it's better to update the commit as shown below.
`feat(android): add support for hiding video duration`

WDYT?

**Review by KrzysztofMoch — APPROVED** [review]

**Pull request merged**